src/test/groovy/org/prebid/server/functional/tests/OrtbConverterSpec.groovy

```diff
@@ -597,14 +598,14 @@ class OrtbConverterSpec extends BaseSpec {
                 podseq = PBSUtils.randomNumber
                 mincpmpersec = PBSUtils.randomDecimal
                 slotinpod = PBSUtils.randomNumber
-                plcmt = PBSUtils.getRandomEnum(VideoPlcmtSubtype)
+                plcmt = placement
             }
         }
 
         when: "Requesting PBS auction with ortb 2.5"
         prebidServerServiceWithElderOrtb.sendAuctionRequest(bidRequest)
 
-        then: "BidResponse shouldn't contain the imp[0].video.* as on request"
+        then: "Bidder request shouldn't contain the imp[0].video.* as on request"
         verifyAll(bidder.getBidderRequest(bidRequest.id)) {
```

Comment:
Please extract the call bidder into a separate variable instead of calling two times.